### PR TITLE
Get rid of asserts in helper classes #2295

### PR DESCRIPTION
**Pull request by anezboretskiy**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
-Refactored all methods like **SelenideElement.shouldBe(Condition)** in **waitUntilScreenReady()** cause the main goal of them is waiting for element's
-Refactored only methods which correspond opening pages 

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorCreateView.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public ConnectorsView setConnectorConfig(String connectName, String configJson)
     }
 
     @Step("Verify that page 'Create Connector' opened")
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorCreateView.java:37]: need to fix step description: 'Wait until page opened'

Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorCreateView.java:37]: @anezboretskiy better just delete description from annotation

Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorCreateView.java]: @VladSenyuta description deleted

**Review by VladSenyuta — APPROVED** [review]

**Comment by Haarolean** [comment]
@anezboretskiy don't forget to link the issues

**Pull request merged**